sphinx/builders/html/__init__.py

```diff
@@ -773,12 +773,12 @@ def onerror(filename: str, error: Exception) -> None:
                        excluded, context=context, renderer=self.templates, onerror=onerror)
 
     def copy_html_logo(self) -> None:
-        if self.config.html_logo:
+        if self.config.html_logo and not urlparse(self.config.html_logo).scheme:
```

Comment:
It would be better to add a helper function like `isurl(s)` for readability. I think `sphinx.util` is a good place to add it.